Tests/SonarScanner.MSBuild.Shim.Test/PropertiesFileGeneratorTests.cs

```diff
@@ -751,18 +723,15 @@ public void GenerateFile_WhenNoGuid_LogWarnings()
             var result = new PropertiesFileGenerator(config, logger).GenerateFile();
 
             // Assert
-            result.RanToCompletion.Should().BeFalse();
```

Comment:
Removed redundant assertion. It is already inside `AssertFailedToCreatePropertiesFiles`